mindsdb/utilities/config.py

```diff
@@ -443,6 +486,12 @@ def raise_warnings(self, logger) -> None:
         if 'log' in self._config:
             logger.warning("The 'log' config option is no longer supported. Use 'logging' instead.")
 
+        if os.environ.get('MINDSDB_HTTP_SERVER_TYPE', '') != '':
+            logger.warning(
+                "Env variable 'MINDSDB_HTTP_SERVER_TYPE' is going to be deprecated soon. "
```

Comment:
Isn't variable name the same here? MINDSDB_HTTP_SERVER_TYPE

Reply:
indeed, fixed